config/config-package.py

```diff
@@ -34,13 +34,13 @@ def copy_with_meta(template_name, destination, config_type, **kw):
     'path', type=pathlib.Path, help='path to the repository to be configured')
 parser.add_argument(
     '--no-commit',
-    dest='no_commit',
-    action='store_true',
+    dest='commit',
+    action='store_false',
```

Comment:
I think it additionally needs `default =True` as its default value is `None`.

Reply:
No, default values are created automatically for `store_true` and `store_false`. See https://docs.python.org/3.7/library/argparse.html#action

Reply:
I have added default values to make it easier to understand the code from eyeballing. As I said, they are not needed unless you want to have a default other than the automatically created `True` or `False`.